schemas/CodiusVarsSpec.json

```diff
@@ -0,0 +1,58 @@
+{
+  "type":"object",
+  "properties":{
+    "vars":{
+      "type":"object",
+      "properties":{
+        "public":{
+          "type":"object",
+          "additionalProperties":{
+            "type":"object",
+            "properties":{
+              "encoding":{
```

Comment:
the hashes of private variables shouldn't be stored in the file here. The private variables in this file should be automatically put into the compiled version

Reply:
That's what I initially suspected. This makes the private variable section of the manifest generation function a bit simpler.